docs/apidiffs/current_vs_latest/opentelemetry-sdk-testing.txt

```diff
@@ -1,2 +1,8 @@
 Comparing source compatibility of  against 
-No changes.
\ No newline at end of file
+***  MODIFIED CLASS: PUBLIC FINAL io.opentelemetry.sdk.testing.assertj.TracesAssert  (not serializable)
+	===  CLASS FILE FORMAT VERSION: 52.0 <- 52.0
+	+++  NEW METHOD: PUBLIC(+) STATIC(+) io.opentelemetry.sdk.testing.assertj.TracesAssert assertThat(java.util.List)
++++  NEW CLASS: PUBLIC(+) io.opentelemetry.sdk.testing.trace.TraceUtil  (not serializable)
+	+++  CLASS FILE FORMAT VERSION: 52.0 <- n.a.
+	+++  NEW SUPERCLASS: java.lang.Object
+	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.util.Comparator SPAN_DATA_COMPARATOR
```

Comment:
I am extremely hesitant to add public constants to our API surface, even in a testing library. Especially since this one is not named in any way to identify what it is for or what how it works. For this PR, let's please not include this piece in the public API. It's simple enough that anyone who needs it can trivially write it, and I'd prefer not to have to support public constants unless there is a very compelling reason for it.

Reply:
ok, will change it.